Server/Server.Exec/MainServer.cpp

```diff
@@ -1,36 +1,28 @@
+
 #include <iostream>
 #include <stdexcept>
 
-#include <Server.hpp>
-#include <ArgumentParser.hpp>
+#include "ServerBuilder.hpp"
+#include "ArgParser.hpp"
+
+using Server::Builder::ServerBuilder;
 
 int main(int argc, const char** argv)
 {
-    using Base::Logger::LogLevel;
-    using Base::Logger::FileLogger;
-
-    uint16_t port;
-
-    try
-    {
-        ArgumentParser parser(argc, argv);
-        port = parser.getPort();
-    }
-    catch (const std::runtime_error& e)
-    {
-        FileLogger::getInstance().log
-        (
-            e.what(), 
-            LogLevel::ERR
-        );
-    }
+    ArgParser parser(argc, argv);
+    auto      server = ServerBuilder()
```

Comment:
Please, explain why in this situation we should use one port? This port for database or server?

Reply:
This is a draft. Work isn't finished.
Also, some details are predefined, and explicit initialization need in case of direct intentions.